src/main/java/com/buuz135/sushigocrafting/client/gui/RollerCraftButtonAddon.java

```diff
@@ -56,7 +56,7 @@ public boolean mouseClicked(double mouseX, double mouseY, int button) {
             Minecraft.getInstance().getSoundManager().play(new SimpleSoundInstance(SoundEvents.UI_BUTTON_CLICK.get(), SoundSource.PLAYERS, 0.2F, 1.0F, Minecraft.getInstance().level.getRandom(), Minecraft.getInstance().player.blockPosition()));
             ILocatable locatable = (ILocatable) ((AbstractContainerScreen) screen).getMenu();
             CompoundTag nbt = new CompoundTag();
-            nbt.putInt("Button", button);
+            nbt.putInt(String.valueOf(Component.translatable("text.sushigocrafting.button")), button);
```

Comment:
This is not a text entry, its just an ID for packets

Reply:
My mistake